requirements.txt

```diff
@@ -9,3 +9,4 @@ oauth2client==4.1.*
 Click==6.*
 numpy==1.*
 six>=1.13.0,<2dev
+pandas>=1.0.5,<2.*
```

Comment:
Major upgrade can contain API changes, patch upgrades mostly contain bugfixes, so that's why later than 1.0.5, but before v2. Are there any issues with specifying restrictive requirements like this?